lab_2/main.py

```diff
@@ -0,0 +1,295 @@
+import requests
+import textwrap
+import csv
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+from pprint import pprint
+from collections import Counter
+
+
+class MovieData:
+    genre_url = 'https://api.themoviedb.org/3/genre/movie/list?language=en'
+    data_url = "https://api.themoviedb.org/3/discover/movie?include_adult=false&include_video=false&sort_by" \
+               "=popularity.desc&page={!!!!}"
+    headers = {
+        "accept": "application/json",
+        "Authorization": "Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIzMTI3NGFmYTRlNTUyMjRjYzRlN2Q0NmNlMTNkOTZjOSIsInN1YiI6IjVkNmZhMWZmNzdjMDFmMDAxMDU5NzQ4OSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.lbpgyXlOXwrbY0mUmP-zQpNAMCw_h-oaudAJB6Cn5c8"
+    }
+
+    def __init__(self):
+        pages = input("Please, enter the number of pages you want to work with: ")
+        self.pages = int(pages)
```

Comment:
move entering and type conversion out of the class.
just pass pages as a parameter